tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -520,11 +520,94 @@ async def with_java_base(context: PipelineContext, build_platform: str, jdk_vers
+    "destination-tidb": "dbt-tidb==1.0.1",
+    "destination-bigquery": "dbt-bigquery==1.0.0",
+}
+
+DESTINATION_SPECIFIC_NORMALIZATION_DOCKERFILE_MAPPING = {
+    **BASE_DESTINATION_SPECIFIC_NORMALIZATION_DOCKERFILE_MAPPING,
+    **{f"{k}-strict-encrypt": v for k, v in BASE_DESTINATION_SPECIFIC_NORMALIZATION_DOCKERFILE_MAPPING.items()},
+}
+
+DESTINATION_SPECIFIC_NORMALIZATION_ADAPTER_MAPPING = {
+    **BASE_DESTINATION_SPECIFIC_NORMALIZATION_ADAPTER_MAPPING,
+    **{f"{k}-strict-encrypt": v for k, v in BASE_DESTINATION_SPECIFIC_NORMALIZATION_ADAPTER_MAPPING.items()},
+}
+
+
+def with_normalization(context: ConnectorContext) -> Container:
+    normalization_directory = context.get_repo_dir("airbyte-integrations/bases/base-normalization")
+    sshtunneling_file = context.get_repo_dir(
+        "airbyte-connector-test-harnesses/acceptance-test-harness/src/main/resources", include="sshtunneling.sh"
+    ).file("sshtunneling.sh")
+    normalization_directory_with_build = normalization_directory.with_new_directory("build")
+    normalization_directory_with_sshtunneling = normalization_directory_with_build.with_file("build/sshtunneling.sh", sshtunneling_file)
+    normalization_dockerfile_name = DESTINATION_SPECIFIC_NORMALIZATION_DOCKERFILE_MAPPING.get(
+        context.connector.technical_name, "Dockerfile"
+    )
+    return normalization_directory_with_sshtunneling.docker_build(normalization_dockerfile_name)
+
+
+async def with_java_base_and_normalization(context: PipelineContext, build_platform: Platform) -> Container:
+    yum_packages_to_install = [
+        "python3",
+        "python3-devel",
+        "jq",
+        "sshpass",
+        "git",
+    ]
+
+    dbt_adapter_package = DESTINATION_SPECIFIC_NORMALIZATION_ADAPTER_MAPPING.get(context.connector.technical_name, "dbt-bigquery==1.0.0")
+
+    pip_cache: CacheVolume = context.dagger_client.cache_volume("pip_cache")
+    java_base = await with_java_base(context, build_platform)
+    return (
+        java_base.with_exec(["yum", "install", "-y"] + yum_packages_to_install)
+        .with_exec(["alternatives", "--install", "/usr/bin/python", "python", "/usr/bin/python3", "60"])
+        .with_mounted_cache("/root/.cache/pip", pip_cache)
+        .with_exec(["python", "-m", "ensurepip", "--upgrade"])
+        .with_exec(["pip3", "install", dbt_adapter_package])
+        .with_directory("airbyte_normalization", with_normalization(context).directory("/airbyte"))
+        .with_workdir("airbyte_normalization")
+        .with_exec(["sh", "-c", "mv * .."])
+        .with_workdir("/airbyte")
+        .with_exec(["rm", "-rf", "airbyte_normalization"])
+        .with_workdir("/airbyte/base_python_structs")
+        .with_exec(["pip3", "install", "."])
+        .with_workdir("/airbyte/normalization_code")
+        .with_exec(["pip3", "install", "."])
+        .with_workdir("/airbyte/normalization_code/dbt-template/")
+        .with_exec(["dbt", "deps"])
+        .with_workdir("/airbyte")
+    )
```

Comment:
Basically copying what's declared in the `destination-bigquery` [Dockerfile](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/destination-bigquery/Dockerfile#L13)